src/controllers/factorycontroller.js

```diff
@@ -105,6 +118,22 @@ class FactoryController extends OrderController
         return true;
     }
 
+    extraTruck (formValues)
+    {
+        if (this.validateTruck(truck)) {
+            this._updateMoney(-GAME.model.config.costExtraTruck);
+            this._updateMaxSimultaneousOrders(GAME.model.config.maxSimultaneousOrders +1);
+        }
+    }
+
+    validateTruck(truck)
+    {
+        if(truckCost>GAME.model.config.money) {
```

Comment:
Spaces around operators, like `truckCost > GAME.model.config.money`. Where is `truckCost` defined?